doc/adr/0009-container-image-tags.md

```diff
@@ -0,0 +1,55 @@
+# 9. Convention for tagging of container images
+
+Date: 2020-03-13
+
+## Status
+
+Proposed
+
+## Context
+
+Libero's products are assembled from product specific, general libero and external container images.
+
+Integration testing and reproducible deploys require uniquely identifiable image tags.
+
+
+## Decision
+
+### Tagging flow
+
+- each container image of a Libero container shall be tagged with the git-sha of the build
```

Comment:
ADRs are usually of limited scope, ideally one sentence as the decision, so that they can be superseded or revisited independently. In practice, we have found that when decisions are strongly related it's more understandable to keep them all together.

Is there any possible splitting of this ADR? For example:
- tagging images of independent projects with git-sha
- per-product version of projects (e.g. `liberoadmin/reviewer-client:reviewer-full_git_hash`). This has consistent tagging with that git-sha version as a consequence. It also is a wider decision that image tagging.
- semantic versioning of products (depends on the one before). Also a wider decision.

Reply:
I would tend towards the latter: keep strongly related decisions together and provide a one stop shop for understanding our image tagging.